code/control/DNRoot.php

```diff
@@ -675,6 +680,7 @@ public function postsnapshotsuccess(SS_HTTPRequest $request) {
 	 * @return \SS_HTTPResponse
 	 */
 	public function project(SS_HTTPRequest $request) {
+		$this->isOverview = true;
```

Comment:
Use `$this->setCurrentActionType(DNRoot::PROJECT_OVERVIEW);` and then use that in the ->Menu call?